api/clients/v2/verification/cert_verifier.go

```diff
@@ -86,14 +94,35 @@ func (cv *CertVerifier) CheckDACert(
 
 	// TODO: determine if there's any merit in passing call options to impose better determinism and
 	// safety on the operation
-	result, err := certVerifierCaller.CheckDACert(
-		&bind.CallOpts{Context: ctx},
-		certBytes,
-	)
+
+	callMsgBytes, err := cv.v2VerifierBinding.TryPackCheckDACert(certBytes)
+	if err != nil {
+		return &CertVerifierInternalError{Msg: "pack checkDACert call", Err: err}
+	}
+
+	// Perform an eth_call with a gas limit of 20M or 2/3 of the current block gas limit.
+	// On-chain (e.g. in a rollup prover tx), budget additional gas since checkDACert adds
+	// to the overall execution path. The 2/3 factor is a heuristic and may be tuned further.
```

Comment:
wouldnt mention 2/3 here, but instead above the constant. 2/3 might become outdated very soon given clients are raising their max gas limit no?